search-commons/src/test/java/no/unit/nva/search/IndexDocumentTest.java

```diff
@@ -70,21 +83,59 @@ public void toIndexDocumentReturnsIndexDocumentWitEmptyContributorListWhenPublic
     @Test
     public void toIndexDocumentReturnsIndexDocumentWithNoDateWithPublicationDateIsNull()
         throws MalformedURLException, InvalidIssnException {
-        Publication publication = PublicationGenerator.publicationWithIdentifier();
+        Publication publication = publicationWithIdentifier();
         publication.getEntityDescription().setDate(null);
         IndexDocument indexDocument = IndexDocument.fromPublication(publication);
         assertThat(indexDocument.getPublicationDate(),is(nullValue()));
     }
 
 
+    @Test
+    void toJsonStringSerializesUrisAsIdentifiers()
```

Comment:
Is this test name correct?

Reply:
Nope!  renamed -> toJsonStringSerializesRequiredFields

Reply:
renamed to toJsonStringSerializesRequiredFields